src/Tools/AI Test Toolkit/src/TestSuite/AITTestSuiteImportExport.XmlPort.al

```diff
@@ -69,8 +110,25 @@ xmlport 149031 "AIT Test Suite Import/Export"
                         AITestMethodLine."Line No." := AITTestMethodLine."Line No." + 10000;
                     end;
                 }
+
+                trigger OnBeforeInsertRecord()
+                begin
+                    if SkipTestSuites.Contains(AITSuite.Code) then
+                        currXMLport.Skip();
+                end;
             }
         }
     }
+
+    trigger OnInitXmlPort()
+    var
+        AITTestContextImpl: Codeunit "AIT Test Context Impl.";
+    begin
+        MD5FileHash := AITTestContextImpl.GetAndClearMD5HashForTheImportedXML();
```

Comment:
We do not need to use static methods, we can pass the codeunits around. You could also have a Get / Set Hash functionality

Reply:
I will update this.